databricks/koalas/frame.py

```diff
@@ -2541,7 +2541,7 @@ def astype(self, dtype) -> 'DataFrame':
         return DataFrame(sdf, self._metadata.copy())
 
     # TODO: percentiles, include, and exclude should be implemented.
-    def describe(self) -> 'DataFrame':
+    def describe(self, percentiles=[0.25, 0.5, 0.75]) -> 'DataFrame':
```

Comment:
Please use `None` for the default value and set it to `[0.25, 0.5, 0.75]` if it's `None` later. A problem mentioned here https://github.com/databricks/koalas/pull/21#discussion_r256520365 might happen.

Reply:
Yea, using a mutable value as a default is actually quite dangerous (see https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html)

Reply:
I think if the default value is immutable, we're safe tho.

Reply:
Done